src/main/java/com/nylas/NeuralOcr.java

```diff
@@ -0,0 +1,32 @@
+package com.nylas;
+
+import java.util.List;
+
+public class NeuralOcr extends File {
+	private List<String> ocr;
```

Comment:
Can you tell if it is possible for there to be more than 1 string returned in this list in the response?  If not, perhaps we should include a convenience accessor for getting just the first element.

Reply:
The API returns a list as every page is it's own array element. So for example if we have a request that processed 3 pages it would return an array of size 3.